defs.h

```diff
@@ -21,7 +21,7 @@
 #define MAX_LVLMTYPES			24
 #define MAX_SPELLS				52
 #define MAX_SPELL_LEVEL			15
-#define SPELLBIT(s) ((__int64)1 << (s - 1))
+#define SPELLBIT(s) ((Uint64)1 << (s - 1))

```

Comment:
`1ULL`?

Reply:
I think we have this in function form, so would suggest dropping the macro.

Reply:
Where?

Reply:
`spells.cpp` `GetSpellBitmask`.